src/main/java/com/zendesk/maxwell/producer/MaxwellKafkaProducer.java

```diff
@@ -97,7 +99,10 @@ private Integer getNumPartitions(String topic) {
 	}
 
 	private String generateTopic(String topic, RowMap r){
```

Comment:
Also does this method need to take `topic` parameter? Can't it rely on `this.topic`?